App/Sagas/LocationSagas.js

```diff
@@ -0,0 +1,17 @@
+import { call, put } from 'redux-saga/effects'
+import { delay } from 'redux-saga'
+import LocationActions from '../Redux/LocationRedux'
+
+// 15 minutes
+const updateDelay = 15 * 60 * 1000
+
+export function* getNearbyUpdates (api, action) {
+  let response = yield call(api.getNearby)
```

Comment:
Not at all! That's a super good point. I was honestly modeling after what was done in the ScheduleUpdate saga, but this makes more sense. 

And now that I'm thinking about it, I'm wondering if the repetition is even necessary. For speaker data, it might be more necessary, but I can't imagine things nearby needing to be refreshed more than each time the app starts up...